podpac/core/managers/aws.py

```diff
@@ -173,6 +181,13 @@ def _function_name_default(self):
 
         return settings["FUNCTION_NAME"]
 
+    @tl.default("function_triggers")
```

Comment:
This block seems to indicate I must set `function_eval_trigger` to 'APIGateway' in order to build the api gateway. More generally, it is not obvious to me how to specify that I want optional resources built (e.g. API Gateway).